cli/src/commands/log.rs

```diff
@@ -118,15 +118,21 @@ pub(crate) fn cmd_log(
     let template = workspace_command.parse_commit_template(&template_string)?;
     let with_content_format = LogContentFormat::new(ui, command.settings())?;
 
+    let commit_node_template =
+        workspace_command.parse_commit_template(&command.settings().commit_node_template())?;
+
+    // TODO(algmyr): Make this not use the commit template language, but isn't that
+    //               basically just a static string?
```

Comment:
Ok, in light on the comments about labels and coloring having this not be a plain string is preferable. I'll make it an empty `GenericTemplateLanguage` for the time being. LMK if this is fine.

Reply:
Sounds good to me. Hopefully we can merge the elided template with the commit one later.

Reply:
> Hopefully we can merge the elided template with the commit one later.

I actually like the idea of splitting them up because there's almost no overlap in what properties they share. Elided nodes are not associated with any particular commit.